driver-core/src/main/com/mongodb/connection/ConnectionPoolSettings.java

```diff
@@ -19,7 +19,9 @@
 import com.mongodb.ConnectionString;
 import com.mongodb.annotations.Immutable;
 import com.mongodb.annotations.NotThreadSafe;
+import com.mongodb.event.ConnectionCreatedEvent;
```

Comment:
Comment: As these imports are just for javadocs I'm surprised checkstyle doesn't complain

Reply:
In some places our static checkers complained (which is wrong in my opinion), but here they fortunately did not.